Backpack/HorizontalNavigation/Classes/BPKHorizontalNavigation.m

```diff
@@ -274,6 +274,12 @@ - (void)repopulateStackview {
     for (id<BPKHorizontalNavigationOptionType> option in self.options) {
         UIControl<BPKHorizontalNavigationItem> *newItem = [self createHorizontalNavigationItemWithDefinition:option];
         [newItem addTarget:self action:@selector(updateSelection:) forControlEvents:UIControlEventTouchUpInside];
+        if (self.selectedColor != nil) {
+            newCell.selectedColor = self.selectedColor;
+        }
+        if (self.size != nil) {
```

Comment:
Thanks for doing this fix! This will always be true as `size` is not nullable.

Reply:
Good catch Geroge 👍

Reply:
Thanks, good catch! 👍